src/styles/index.scss

```diff
@@ -3 +1,54 @@
+@import './abstract/constants';
+@import './abstract/placeholders';
+@import './base/base';
+
+.calendar {
+  display: flex;
+  padding: $calendar-indent;
+  flex-wrap: wrap;
+  align-items: center;
```

Comment:
The align-items: center; property seems unnecessary here since flex items are already in a row direction and their height is not defined by the parent. (Code Quality Issue)